specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/locations.json

```diff
@@ -270,23 +275,35 @@
           "format": "int64"
         },
         "cores_available": {
-          "description": "The number of courses available in the region.",
+          "description": "The number of cores available in the region.",
           "type": "integer",
           "format": "int64"
         }
-      }
+      },
+      "readOnly": true
     },
     "QuotaCapability": {
       "description": "The regional quota capability.",
       "properties": {
+        "cores_used": {
+          "description": "The number of cores used in the subscription.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "max_cores_allowed": {
```

Comment:
Please ensure you address this naming in your next API version

Reply:
> Please ensure you address this naming in your next API version

Yes. I have created a issue https://github.com/Azure/azure-rest-api-specs/issues/6724